owners-bot/.eslintrc

```diff
@@ -0,0 +1,103 @@
+{
+  "parser": "babel-eslint",
```

Comment:
I think this needs an entry here:
https://github.com/ampproject/amp-github-apps/blob/f43228a5d7db38ad4742d5a981158deae46930df/package.json#L18-L24

Reply:
If it works inside `owners/package.json`, that'll be better for this project; otherwise, what Raghu said